### ISSUE TITLE
k-navbar - Universal design problem / VoiceOver

### ISSUE THREAD

**Issue by thomasekdahlN** [konstaui/konsta#68]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

App mypostnord

### Bug description

When the html code for k-navbar is generated, an extra div is added at the top of the code that is printed. This is marked below, and begins with the class absolute.

<div class="w-full z-20 top-0 pt-safe sticky text-navbar-ios">
<div class="absolute w-full h-full left-0 top-0 bg-primary dark:bg-light-black hairline-b"></div>
<div class="flex relative items-center w-full overflow-hidden pl-2-safe pr-2-safe h-11 justify-between">
…
Because of this, the return link does not work when you use voice over on the phone. It only reads out the title.

### Expected Behavior

Navigation elements should be clickable when the phone is using VoiceOver mode on mobile phone

### Actual Behavior

Not possible to click on navigation links in VoiceOver mode on mobile phone

### Konsta UI version

v0.11.1

### Platform/Target and Browser Versions

iOS

### Validations

- [X] Follow our [Code of Conduct](https://github.com/konstaui/konsta/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://konstaui.com).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Konsta UI issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by thomasekdahlN** [comment]
Is it possible to pay to have this issue solved?

**Comment by nolimits4web** [comment]
Can you give an example of how the HTML should be modified that would work correctly?

**Comment by thomasekdahlN** [comment]
On a phone with voice over, the title was the only element that was read. It was not possible to click an svg link element used as a back link. We solved this by making the back link to a back button.
On voice over, a button is always read as an button. A link need a href to be read as a link.


Thomas Ekdahl
Privat
Mobile: +47 911 43 630
